worker/respec-highlight.js

```diff
@@ -0,0 +1,22 @@
+/* eslint sort-imports: "off" */
+import highlight from "highlight.js/lib/core";
+import abnf from "highlight.js/lib/languages/abnf";
+import css from "highlight.js/lib/languages/css";
+import http from "highlight.js/lib/languages/http";
+import javascript from "highlight.js/lib/languages/javascript";
+import json from "highlight.js/lib/languages/json";
+import xml from "highlight.js/lib/languages/xml";
+
+highlight.configure({
+  tabReplace: "  ", // 2 spaces
+  languages: ["abnf", "css", "http", "javascript", "json", "xml"],
+});
+
+highlight.registerLanguage("abnf", abnf);
+highlight.registerLanguage("css", css);
+highlight.registerLanguage("http", http);
+highlight.registerLanguage("javascript", javascript);
+highlight.registerLanguage("json", json);
+highlight.registerLanguage("xml", xml);
+
```

Comment:
```suggestion
const langMap = new Map([
  ["abnf", abnf], 
  ["css", css],
  ["http", http],
  ["javascript", javascript],
  ["json", json],
  [ "xml", xml],
])

highlight.configure({
  tabReplace: "  ", // 2 spaces
  languages:  [...langMap.keys()],
});

for(const [key, highlighter] of langMap.entries()) {
  highlight.registerLanguage(key, highlighter);
}

```

Reply:
I think the former is straight-forward. Other requires more brain processing.
Opinions:tm: